src/_transcripts/64.json

```diff
@@ -0,0 +1,1820 @@
+{
+  "speakers": {
+    "spk_0": "spk_0",
+    "spk_1": "spk_1"
+  },
+  "segments": [
+    {
+      "speakerLabel": "spk_0",
+      "start": 0,
+      "end": 4.24,
+      "text": " Unless you have been living under a rock, you probably notice that Rust is gaining more and"
+    },
+    {
+      "speakerLabel": "spk_0",
+      "start": 4.24,
+      "end": 9.36,
+      "text": " more traction by the day. So today we want to talk about writing Lambda functions in Rust."
+    },
+    {
+      "speakerLabel": "spk_0",
+      "start": 9.36,
+      "end": 13.44,
+      "text": " We will be discussing why you might want to do something like this, analyzing pros and cons,"
+    },
+    {
+      "speakerLabel": "spk_0",
+      "start": 13.44,
+      "end": 18.56,
+      "text": " then we will look at the steps needed to author and publish your very first Lambda in Rust."
+    },
+    {
+      "speakerLabel": "spk_0",
+      "start": 18.56,
+      "end": 22.64,
+      "text": " Finally, we will give our opinion on what's going to be the future of Rust in Lambda,"
+    },
+    {
+      "speakerLabel": "spk_0",
+      "start": 22.64,
+      "end": 24.48,
+      "text": " and why we are so excited about it."
+    },
+    {
+      "speakerLabel": "spk_0",
+      "start": 24.48,
+      "end": 29.84,
+      "text": " My name is Usano and today I'm joined by Owen and this is AWS Bytes podcast."
```

Comment:
Our substitution vocabulary is not working. Perhaps we have malformed JSON again? Needs checking!